README.md

```diff
@@ -9,20 +9,25 @@ many pieces. It can drive a very wide range of build and packaging systems,
 so that you can simply list all the upstream projects you want and have
 them built and installed together as a single tree.
 
+![Snapcraft Overview][overview-image]
+
 For example, say you want to make a product that includes PyPI packages,
 Node.js packages from NPM, Java, and a bunch of daemons written in C and
 C++ that are built with autotools, snapcraft would make assembling the
 final tree very easy.
 
 Snapcraft allows easy crafting of snap packages for the Ubuntu Core
-transactional system.
+transactional system (Snappy).
```

Comment:
better to use the term...  *for the snappy Ubuntu Core transactional system.*

(casing of snappy as seen)